### PR TITLE
tlsr82/backtrace: tc32 backtrace bug fix

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary

- tc32/Make.defs: fix build break after merge unblock/block_task

## Impact

telink tlsr82

## Testing

Internal testing

### CODE REVIEW

File: arch/arm/src/tlsr82/tc32/tc32_backtrace.c [hunk 1]
```diff
@@ -185,25 +201,53 @@ static void *backtrace_push_internal(void **psp, void **ppc)
...
-          found = true;
+          case 0:
+            if (INSTR_IS(ins16, T_SUB_SP_16))
+              {
+                /* Low 7 bit (imm7) is the number (4*imm7) sub to sp */
+
+                frame += (ins16 & 0x7f);
+                state  = 1;
+                break;
+              }
```
Comment (pkarashchenko) [inline arch/arm/src/tlsr82/tc32/tc32_backtrace.c:230]: is this intentional fall-thru?

File: arch/arm/src/tlsr82/tc32/tc32_backtrace.c [hunk 2]
```diff
@@ -63,6 +63,22 @@
 
 static void **g_backtrace_code_regions;
 
+/****************************************************************************
+ * Inline functions
+ ****************************************************************************/
+
+static inline uint32_t tc32_getsp(void)
```
Comment (pkarashchenko) [inline arch/arm/src/tlsr82/tc32/tc32_backtrace.c:70]: why not `__builtin_frame_address(0)`?

File: arch/arm/src/tlsr82/tc32/tc32_backtrace.c (hunk 1 above)
Comment (CV-Bowen) [reply arch/arm/src/tlsr82/tc32/tc32_backtrace.c:230]: Yes, seems add a comment here is better.

File: arch/arm/src/tlsr82/tc32/tc32_backtrace.c (hunk 2 above)
Comment (CV-Bowen) [reply arch/arm/src/tlsr82/tc32/tc32_backtrace.c:70]: I will verify this later.

**Review by acassis — APPROVED** [review]

**Pull request merged**